x-pack/legacy/plugins/reporting/server/lib/esqueue/worker.js

```diff
@@ -273,13 +278,16 @@ export class Worker extends events.EventEmitter {
         if_primary_term: job._primary_term,
         body: { doc }
       })
-        .then(() => {
+        .then((response) => {
           const eventOutput = {
             job: formatJobObject(job),
             output: docOutput,
           };
-
           this.emit(constants.EVENT_WORKER_COMPLETE, eventOutput);
+
+          const formattedDocPath = `/${response._index}/${response._type}/${response._id}`;
```

Comment:
is this a file path? maybe using node's `path.join` would be better if thats the case

Reply:
It is the ID to the document in Elasticsearch. It doesn't depend on OS.

For example, this will log something like:
```
server    log   [12:00:58.906] [info][esqueue][reporting][worker] jzd1sy7n0h549d00621yfa7p - Job data saved successfully: /.reporting-2019.08.11/_doc/jzd1tkyy0h549d0062a56le7
```

Then someone could do: 

    curl -u elastic http://localhost:9200/.reporting-2019.08.11/_doc/jzd1tkyy0h549d0062a56le7

if they needed to check what was saved into Elasticsearch for this particular reporting job.